iphone/Classes/TiUtils.h

```diff
@@ -730,4 +730,6 @@ typedef enum {
  */
 + (UIImage *)imageWithColor:(UIColor *)color;
 
++ (NSString*)stackFromException:(TiContextRef)context exception:(TiValueRef)exception;
```

Comment:
Please format the source with `clang-format -style=file -i Classes/TiUtils.*` and for other related files. Only necessary at the end :-).